docs/content/preview/drivers-orms/orms/go/ysql-gorm.md

```diff
@@ -51,28 +38,32 @@ Clone the Yugabyte [`orm-examples` repository](https://github.com/yugabyte/orm-e
 ```sh
 $ git clone https://github.com/YugabyteDB-Samples/orm-examples.git
 ```
+## Set up the application and install dependencies
 
-Run the following `export` command to specify the `GOPATH` environment variable.
+Initialize Go within the project:
 
 ```sh
-export GOPATH=$GOPATH:$HOME/orm-examples/golang/gorm
+cd golang/gorm
+go mod init gorm-example
 ```
 
-## Build and run the application
-
-Change to the `gorm` directory.
+Download the included packages and dependencies:
 
 ```sh
-$ cd ./golang/gorm
+go mod tidy
 ```
 
+## Build and run the application
+
 Create the `ysql_gorm` database in YugabyteDB by running the following `ysqlsh` command from the YugabyteDB home directory.
 
 ```sh
 $ ./bin/ysqlsh -c "CREATE DATABASE ysql_gorm"
 ```
 
-Build and start the REST API server by running the following shell script.
+Build and start the REST API server by running the following shell script. The application uses `yugabytedb/pgx` driver, so load balance feature is enabled.
+
+Note: To use the upstream driver replace `github.com/yugabyte/gorm-yugabytedb` with `gorm.io/driver/postgres` in the application code.
```

Comment:
```suggestion
Build and start the REST API server by running the following shell script. The application uses the YugabyteDB smart driver for Go (`yugabytedb/pgx`), which provides the connection load balancing feature.

Note: The application also works with the upstream Go driver (`jackc/pgx`). To use it, simply replace `github.com/yugabyte/gorm-yugabytedb` with `gorm.io/driver/postgres` in the application code.
```

Reply:
Build and start the REST API server by running the following shell script. The application uses the [YugabyteDB smart driver for Go](../../../go/yb-pgx/), which provides the connection load balancing feature.

Note: The application also works with the upstream Go driver (`jackc/pgx`). To use it, replace `github.com/yugabyte/gorm-yugabytedb` with `gorm.io/driver/postgres` in the application code.